batavia/types/Dict.js

```diff
@@ -4,305 +4,11 @@
 
 batavia.types.Dict = function() {
     function Dict(args, kwargs) {
-        Object.call(this);
-        if (args) {
-            this.update(args);
-        }
+        batavia.types.JSDict.call(this, args, kwargs);
     }
 
-    Dict.prototype = Object.create(Object.prototype);
+    Dict.prototype = Object.create(batavia.types.JSDict.prototype);
```

Comment:
So if I'm reading this right, this patch doesn't actually *do* any of that "python compliant" stuff yet - it's just a placeholder for the fact that it's going to be needed in the not-too-distant future. Is that correct?